crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -730,6 +730,12 @@ impl CheckpointBuilder {
                     )
                     .await?;
 
+                self.accumulator.accumulate_checkpoint(
+                    effects.clone(),
+                    sequence_number,
+                    self.epoch_store.clone(),
+                )?;
+
                 let committee = system_state_obj.get_current_epoch_committee().committee;
                 let root_state_digest = Some(
```

Comment:
Should we put this into EndOfEpochData?

Reply:
Put what? The accumulated checkpoint? That is not committed to. The accumulated epoch is, and that is already in EndOfEpochData. I'll remove these accumulator changes to make things less confusing